app/controllers/import/base_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Import::BaseController < ApplicationController
+
+  private
+
+  def get_or_create_namespace
+    existing_namespace = Namespace.find_by("path = ? OR name = ?", @target_namespace, @target_namespace)
```

Comment:
Line is too long. [104/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.